src/cmd/yacc.c

```diff
@@ -1177,19 +1177,23 @@ setup(int argc, char *argv[])
 {
 	long c, t;
 	int i, j, fd, lev, ty, ytab, *p;
-	int vflag, dflag, stem;
+	int vflag, dflag, fdflag, stem;
 	char actnm[8], *stemc, *s, dirbuf[128];
 	Biobuf *fout;
 
 	ytab = 0;
 	vflag = 0;
 	dflag = 0;
+	fdflag = 0 != 0;
 	stem = 0;
 	stemc = "y";
 	foutput = 0;
 	fdefine = 0;
 	fdebug = 0;
 	ARGBEGIN{
+	case 'f':
+		fdflag = 0 == 0;
```

Comment:
Surely just, `fdflag = 0;` ?

Reply:
(er, 1. But still, the boolean is a little too cute.)

Reply:
@dancrossnyc I think it would be best to do something like `#define True 1` (analogously for the other boolean value). Can you tell me if it should be `TRUE`, `True`, `true`, or just `1` directly?

Reply:
(I will remove this particular commit in a future force-push, I am asking for other commits.)

Reply:
For what it is worth, `True` and `TRUE` are used in that way in other places in p9p; and C++ and Go use `true`.

Reply:
<stdbool.h> also `define`s the lower case versions.

Reply:
I went with False and True enumeration constants.